app/controllers/admin/problems_controller.rb

```diff
@@ -1,4 +1,74 @@
 # frozen_string_literal: true
 
 class Admin::ProblemsController < Admin::ApplicationController
+  before_action :set_session
+  before_action :set_mission
+  before_action :set_problem, only: %i[show]
+
+  def new
+    @problem = @mission.problems.new
+  end
+
+  def create
+    @problem = Problem.new(problem_params)
+    unless @problem.save
+      logger.info problem_errors: @problem.errors.full_messages
+      flash[:alert] = "fail to create problem #{@problem.errors.full_messages.join(' ')}"
+      render :new
+    end
+
+    test_command_list = @mission.test_commands
+    tests = Test.where("mission_id = ?", params[:mission_id])
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.